docs/shared/migration/adding-to-existing-project.md

```diff
@@ -322,21 +302,30 @@ To connect to Nx Cloud:
 - Commit and push your changes
 - Go to [https://cloud.nx.app](https://cloud.nx.app), create an account, and connect your repository
 
-#### Connect to Nx Cloud Manually
-
-If you are not able to connect via the automated process at [https://cloud.nx.app](https://cloud.nx.app), you can connect your workspace manually by running:
-
 ```shell
 npx nx connect
 ```
 
-You will then need to merge your changes and connect to your workspace on [https://cloud.nx.app](https://cloud.nx.app).
+When you are prompted to choose your CI provider, choose `GitHub Actions`.
+
+Once the script is finished, it will print a link in the terminal to register your repository in your [Nx Cloud](https://cloud.nx.app) account. Click the link and follow the steps provided to make sure Nx Cloud is enabled on the main branch of your repository.
+
+### Configure Your CI Workflow
+
+The `nx connect` command you ran in the previous step created a `.github/workflows/ci.yml` file that contains a CI pipeline that will run the `lint`, `test`, `build` and `e2e` tasks for projects that are affected by any given PR.
```

Comment:
`nx connect` does not generate workflow file since the onboarding from Cloud only adds token, and it's weird to have two PRs/commits (one for token, one for workflow file) being done at the same time. Let's get users to run `nx g ci-workflow` and choose `Github Actions`.

(same for other standalone tutorials)